src/app/shared/search/advanced-search/advanced-search.component.ts

```diff
@@ -0,0 +1,150 @@
+import { Component, Inject, Input, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { slide } from '../../animations/slide';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { FormBuilder } from '@angular/forms';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { SearchService } from '../../../core/shared/search/search.service';
+import { RemoteData } from '../../../core/data/remote-data';
+import { SearchFilterConfig } from '../models/search-filter-config.model';
+import { SearchConfigurationService } from '../../../core/shared/search/search-configuration.service';
+import { SearchFilterService } from '../../../core/shared/search/search-filter.service';
+import { SEARCH_CONFIG_SERVICE } from '../../../my-dspace-page/my-dspace-page.component';
+import { RemoteDataBuildService } from '../../../core/cache/builders/remote-data-build.service';
+import { PaginatedSearchOptions } from '../models/paginated-search-options.model';
+import { AppConfig, APP_CONFIG } from 'src/config/app-config.interface';
+@Component({
+  selector: 'ds-advanced-search',
+  templateUrl: './advanced-search.component.html',
+  styleUrls: ['./advanced-search.component.scss'],
+  animations: [slide],
+})
+
+
+export class AdvancedSearchComponent implements OnInit {
+  /**
+    * An observable containing configuration about which filters are shown and how they are shown
+    */
+  @Input() filters: Observable<RemoteData<SearchFilterConfig[]>>;
+  @Input() searchOptions: PaginatedSearchOptions;
+  /**
+   * List of all filters that are currently active with their value set to null.
+   * Used to reset all filters at once
+   */
+  clearParams;
+
+  /**
+   * The configuration to use for the search options
+   */
+  @Input() currentConfiguration;
+
+  /**
+   * The current search scope
+   */
+  @Input() currentScope: string;
+
+  /**
+   * True when the search component should show results on the current page
+   */
+  @Input() inPlaceSearch;
+
+  /**
+   * Emits when the search filters values may be stale, and so they must be refreshed.
+   */
+  @Input() refreshFilters: BehaviorSubject<boolean>;
+
+  /**
+   * Link to the search page
+   */
+  currentURL: string;
+  notab: boolean;
+  @Input() searchConfig;
```

Comment:
@GauravD2t

this seems to not be used, if so could you remove it?

Reply:
remove  currentURL: string;